### PR TITLE
add docs for school quote

### PR DESCRIPTION
**Pull request by variousauthors**
**JIRA**: https://edvisorio.atlassian.net/browse/EDM-979

This PR adds documentation for the school quote and school quote list queries.

**QA Steps:**
 - read and enjoy the new docs section
 - you should feel confident that you could implement an enterprise software system on the back of this API

### CODE REVIEW

File: source/includes/_school-api-guide.md [hunk 1]
```diff
@@ -2,4 +2,417 @@
 
 Detailed explanation and examples to using Edvisor Api as a School
 
-## TO DO
+## Web Quotes Engine
+
+Blah blah blah
```
Comment (mila-mamat) [inline source/includes/_school-api-guide.md]: is this part going to be updated in another task soon?

Comment (variousauthors) [reply source/includes/_school-api-guide.md]: Awooooops thanks Mila!

**Review by mila-mamat — APPROVED** [review]
easy to read and understand

**Review by chernandezbl — APPROVED** [review]
I wasn't able to setup the environment so I couldn't QA :(

On the other hand the code looks good: 

- [x] Query for schools are correct
- [x] Descriptions are awesome, especially for union types which commonly cause problems to new developers to GraphQL 👍

**Review by dandc22 — APPROVED** [review]
QA and working 👍

**Review by gabriel-schmoeller — APPROVED** [review]

**Pull request merged**